### ISSUE TITLE
Just uploaded model does not show in model list

### ISSUE THREAD

**Issue by Jonasvsl** [nens/threedi-api-qgis-client#75]
* threedi-api-qgis-client version: 1.5.0.b
* Python version:
* Operating System:

### Description

the model list is not being refreshed after you press change to change model
this is an issue when you have uploaded a new model. the current workflow would be to log out and log in again.
maybe it would be good to give users the option to check something that says 'refresh list from server' 

Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.

### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

**Comment by DamnBack** [comment]
Fixed in the version 2.0.0

**Comment by Jonasvsl** [comment]
@DamnBack why did you reopen this ticket?
